libs/markets/src/lib/components/last-24h-price-change/last-24h-price-change.tsx

```diff
@@ -37,6 +39,28 @@ export const Last24hPriceChange = ({
   const change = priceChange(candles);
   const changePercentage = priceChangePercentage(candles);
 
+  if (orientation === 'vertical') {
```

Comment:
Dont love this. Could probably be done better by just flipping between 'flex-row' and 'flex-col'? You might need to re-arrange the markup though

Reply:
they do look slightly different, it's not just the orientation

I can make another component 

<img width="110" alt="Screenshot 2024-04-26 at 12 57 40" src="https://github.com/vegaprotocol/frontend-monorepo/assets/16125548/91c83d06-42b4-44a4-bf75-8418fcc7cea8">

<img width="72" alt="Screenshot 2024-04-26 at 12 57 31" src="https://github.com/vegaprotocol/frontend-monorepo/assets/16125548/243509dd-b59a-4dd8-b9ca-580d77b6e789">